src/liballoc/boxed.rs

```diff
@@ -511,3 +512,53 @@ impl<'a,A,R> FnOnce<A> for Box<FnBox<A,Output=R>+Send+'a> {
 }
 
 impl<T: ?Sized+Unsize<U>, U: ?Sized> CoerceUnsized<Box<U>> for Box<T> {}
+
+impl<T: Clone> Clone for Box<[T]> {
```

Comment:
We like to track when impls like this were added in case we do start tracking impl stability one day.